.travis.yml

```diff
@@ -13,7 +13,7 @@ install: "pip install -r requirements.txt"
 before_script:
   - python2 testing/test-health-checks.py
   - ssh-keygen -N '' -f ~/.ssh/id_rsa && eval $(ssh-agent) && ssh-add ~/.ssh/id_rsa
-  - ln -sf $ANSIBLE_PLAYBOOK terraform.yml
+  - ln -sf $ANSIBLE_PLAYBOOK mantl.yml
```

Comment:
does this actually get used anywhere? it looks like build-cluster.py just runs "sample.yml".